starwars/src/App.js

```diff
@@ -1,14 +1,41 @@
 import React, { Component } from 'react';
 import './App.css';
+import Character from "./components/Character"
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      starwarsChars: []
+      starwarsChars: [],
+      open: false ,
```

Comment:
By placing open here you are saying you want all of the cards to open at the same time. If you want each card to open when only its button is pressed consider changing the card to a class and adding this variable to `this.state`.